force-app/main/default/classes/FeedComment_TriggerHandler.cls

```diff
@@ -0,0 +1,98 @@
+/**                     **************                    ***
+ * @description       : Helper class for FeedComment Trigger
+ * @author            : Deepak Prajapati (d.prajapati@concret.io)
+ * @last modified on  : 05-10-2021
+ * @last modified by  : Deepak Prajapati (d.prajapati@concret.io)
+**************************************************************/
+public with sharing class FeedComment_TriggerHandler {
+
+    Public static final String ORG_URL = URL.getSalesforceBaseUrl().toExternalForm();
+    
+    /* TEST VARIABLE */
+    @TestVisible public static List<String> testEmails;
+
+    /****************
+    * @description Method for sending email to Account Followers
+    * @author Deepak Prajapati (d.prajapati@concret.io) | 28-09-2021 
+    * @param List<FeedItem> feedItemList --> Trigger.New  
+    * @param String operationType  --> Either Insert or Update
+    ***************************************/
+    public static void FeedComment_Trigger(List<FeedComment> feedComments, String operationType) {
```

Comment:
the method name should be relative right? FeedComment_Trigger is not specifying anythis